docs/UserGuide.md

```diff
@@ -45,6 +45,8 @@ SOCket is a **desktop app for NUS Software Engineering Students to manage the co
 
 **:information_source: Notes about the command format:**<br>
 
+* `INDEX` must be a positive number starting from 1 and ending with corresponding number of Person in SOCket. e.g. `1,2,...`
```

Comment:
Perhaps specify `integer` in particular, can refer to the lines removed in the old version